fix(RELEASE-1769): multi-components using always the same base image

theflockers opened the pull request:
this PR fixes the case when the from_index is passed with the incorrect value instead of the one discovered by the prepare-fbc-release task.

## Describe your changes

## Relevant Jira

## Checklist before requesting a review
- [ ] I have marked as draft or added `do not merge` label if there's a dependency PR
  - If you want reviews on your draft PR, you can add reviewers or add the `release-service-maintainers` handle if you are unsure who to tag
- [ ] My commit message includes `Signed-off-by: My name <email>`
- [ ] I read CONTRIBUTING.MD and [commit formatting](CONTRIBUTING.md#commit-message-formatting-and-standards)
- [ ] I have run the README.md generator script in `.github/scripts/readme_generator.sh` and verified the results using `.github/scripts/check_readme.sh`

johnbieren commented on tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml:
```diff
@@ -199,8 +199,9 @@ spec:
-        # default target_index
+        # default from_index and target_index
         target_index=$(params.targetIndex)
+        from_index=$(params.fromIndex)
```
Isn't this never used? It looks like line 242 will always overwrite this, and it isn't used before line 242 AFAICT

theflockers replied:
That is true. I will remove it.

arewm commented on tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml:
```diff
@@ -238,6 +238,7 @@ spec:
         for((i=0; i<LENGTH; i++)); do
           ocp_version=$(jq -cr --argjson i "$i" '.components[$i].ocpVersion' "${SNAPSHOT_PATH}")
```
Do you plan to use this for anything or it is just used to report the version in the result artifact?

theflockers replied:
The version of it is set in the `prepare-fbc-release`. This is used later by the `publish-fbc-release` to determine if it should do just a single publishing or several.

arewm reviewed (dismissed):
Thanks!

seanconroy2021 reviewed (dismissed):
lgtm

johnbieren commented on tasks/managed/add-fbc-contribution/tests/test-add-fbc-contribution-hotfix.yaml:
```diff
@@ -225,11 +225,6 @@ spec:
                 echo "service account does not match"
               fi
 
-              if [ "$(jq -r '.fromIndex' <<< "${requestParams}")" != "quay.io/scoheb/fbc-index-testing:latest" ]; then
```
probably better to add a fromIndex value around line 70 and keep this check, no? (same for all test files)

theflockers committed:
fix(RELEASE-1769): multi-components using always the same base image

this PR fixes the case when the from_index is passed with
the incorrect value instead of the one discovered by the
prepare-fbc-release task.

Signed-off-by: Leandro Mendes <lmendes@redhat.com>
